presto-docs/src/main/sphinx/release/release-0.245.rst

```diff
@@ -0,0 +1,34 @@
+=============
+Release 0.245
+=============
+
+**Highlights**
+==============
+* New `ENUM_KEY` UDF to get the key for an enum value.
+* Configurable query result compression.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix LambdaDefinitionExpression canonicalization to correctly canonicalize Block constant (:issue:`15424`).
+* Fix enum deserialization bug in map keys.
```

Comment:
what kind of bug?  Would queries fail?  Need a user facing description of what didn't work that now does.

Reply:
@daniel-ohayon can you help describe what the bug was? How would a user experience it?

Reply:
Here is a clearer definition: `Fix a serialization bug which caused maps with string-valued enum keys to be displayed as illegible strings in the Presto CLI output`

Note that enums have just been introduced into the presto core and there is no open-source connector supplying them, so immediate user impact should be non-existant at this point.